net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_bfd_peers.xml

```diff
@@ -0,0 +1,231 @@
+	<aftersaveredirect>pkg.php?xml=frr/frr_bfd_peers.xml</aftersaveredirect>
+	<tabs>	
+		<tab>
+			<text>BFD</text>
+			<url>pkg_edit.php?xml=frr/frr_bfd.xml</url>
+		</tab>
+		<tab>
+			<text>Peers</text>
+			<url>pkg.php?xml=frr/frr_bfd_peers.xml</url>
+			<active/>
+		</tab>	
+		<tab>
+			<text>[Global Settings]</text>
+			<url>pkg_edit.php?xml=frr.xml</url>
+		</tab>
+		<tab>
+			<text>[BGP]</text>
+			<url>pkg_edit.php?xml=frr/frr_bgp.xml</url>
+		</tab>
+		<tab>
+			<text>[OSPF]</text>
+			<url>pkg_edit.php?xml=frr/frr_ospf.xml</url>
+		</tab>
+		<tab>
+			<text>[OSPF6]</text>
+			<url>pkg_edit.php?xml=frr/frr_ospf6.xml</url>
+		</tab>
+		<tab>
+			<text>Status</text>
+			<url>/status_frr.php?protocol=bfd</url>
+		</tab>
+	</tabs>
+	<adddeleteeditpagefields>
+		<columnitem>
+			<fielddescr>Peer Address</fielddescr>
+			<fieldname>peer</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Description</fielddescr>
+			<fieldname>descr</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Label</fielddescr>
+			<fieldname>label</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Shutdown</fielddescr>
+			<fieldname>shutdown</fieldname>
+		</columnitem>
+	</adddeleteeditpagefields>
+	<fields>		
+		<field>
+			<name>Peer Configurations</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Peer Address</fielddescr>
+			<fieldname>peer</fieldname>
+			<description>
+				<![CDATA[IP address of a peer]]>
+			</description>
+			<type>input</type>
+			<size>20</size>
+			<required/>
+		</field>
+		<field>
+			<fielddescr>Description</fielddescr>
+			<fieldname>descr</fieldname>
+			<size>50</size>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>Label</fielddescr>
+			<fieldname>label</fieldname>
+			<description>
+				<![CDATA[
+				Label a peer with the provided word. This word can be referenced later on other daemons to refer to a specific peer.
+				]]>
+			</description>
+			<size>50</size>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>Multihop</fielddescr>
+			<fieldname>multihop</fieldname>
+			<description>
+				<![CDATA[
+				 Expect packets with TTL less than 254 due to more than one hop between peer addresses and listen on the multihop port 4784.
+				 When using multi-hop mode echo-mode will not work, see RFC 5883 section 3.
+				]]>
+			</description>
+			<type>checkbox</type>
+		</field>
+		<field>
+			<fielddescr>Shutdown</fielddescr>
+			<fieldname>shutdown</fieldname>
+			<description>
+				<![CDATA[
+				Enables or disables the peer.
+				When the peer is disabled an "administrative down" message is sent to the remote peer.
+				]]>
+			</description>
+			<type>checkbox</type>
+		</field>
+
+		<field>
+			<name>Source Address/Interface</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Interface</fielddescr>
+			<fieldname>interface</fieldname>
+			<description>Select the interface to source BFD packets from </description>
```

Comment:
There is an extra space at the end of this sentence, also not that it's a huge problem, but ending in a preposition isn't ideal.
"Select the interface which BFD will use as a packet source" is one good way around that.

Reply:
No problem, done.